src/features/lightspeed/inlineSuggestions.ts

```diff
@@ -819,37 +819,33 @@ export async function inlineSuggestionReplaceMarker(position: vscode.Position) {
     `[inline-suggestions] Inline Suggestion Marker Handler triggered using command at ${position.line}`,
   );
 
-  if (
-    lightSpeedManager.settingsManager.settings.lightSpeedService
-      .disableContentSuggestionHeader
-  ) {
-    // Get the current text
-    const line = position.line;
+  // Get the current text
+  const line = position.line;
+
+  const text = editor.document.getText(
+    new vscode.Range(
+      new vscode.Position(line, 0),
+      new vscode.Position(line + 1, 0),
+    ),
+  );
+  // Remove the prepended text
+  const commentPosition = text.indexOf(
```

Comment:
I don't understand what is this `commentPosition` variable for.

Reply:
Though these are shown as new codes, they are actually just re-formatted ones (encluded in an if-block, which was removed with this PR).

I understand the variable is used just to check if the pre-defined comment exists or not.